docs/_docs/reference/changed-features/structural-types.md

```diff
@@ -32,30 +32,48 @@ configure how fields and methods should be resolved.
 
 ## Example
 
-Here's an example of a structural type `Person`:
+Before we can use structural types, we need a subclass of `Selectable` that overrides `selectDynamic` if we want `a.b` and `applyDynamic` if we want `a.b(c)` (more details further down). As an example, here is a class `Record`, that only provides `selectDynamic`:
```

Comment:
I think this language isn't very precise. For  _selection_ from a structural type, the type needs to subtype `Selectable`. However structural refinements can be to any type